website/docs/concepts/system_overview/README.md

```diff
@@ -75,3 +73,40 @@ Below is the architectural diagram that visually represents how these services i
 This introduction is designed to be a primer.
 Each service has its depths and intricacies, which you'll uncover as you dive deeper into Amiga development.
 Enjoy the journey!
+
+## Open Source Code Examples
+
+Diving into Amiga robot development is made significantly easier with our collection of over
+30 open source Python code examples.
+These examples are meticulously designed to cater to a wide range of skill levels, from beginners
+to advanced developers, ensuring that everyone can find a suitable starting point for their journey.
+By exploring these examples, you'll gain hands-on experience with our API, learning how to effectively
+communicate with and leverage the robot's diverse sensors.
+
+### Where to Run the Examples
+
+Every code example comes with detailed instructions on how to run it, tailored to the specifics of
+the example.
+A key feature of our development environment is its flexibility; examples can be executed on your
+local machine as well as directly on the Amiga's brain.
+This adaptability is essential for developers who wish to work within their own environments or
+need to manage operations across multiple robots.
+
+#### Running Examples on Your Local Machine
+
+To run examples from your local machine and communicate with the Amiga robot over Wifi, you must
+modify the `service_config.json` file (or its equivalent).
+Specifically, the `host` field should be changed from `localhost` to your robot's unique tailscale name
```

Comment:
Add a comment about + link to the instructions for configuring tailscale

Reply:
Good suggestion. Implemented here: 0e04b7a884259bc3c9baa37585235f275e6abd57